src/components/global/nav-bar/nav-bar.vue

```diff
@@ -0,0 +1,103 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import MonteCinemaIcon from '../../../assets/icons/monte-cinema-logo.svg';
+    import ToggleNavOpenIcon from '../../../assets/icons/toggle-nav-open-icon.svg';
+    import ToggleNavClosedIcon from '../../../assets/icons/toggle-nav-closed-icon.svg';
+    import NavLinks from './nav-links.vue';
+    import NavButtonContainer from './nav-button-container.vue';
+
+    export default defineComponent({
+        name: 'NavBar',
+        components: {
+            MonteCinemaIcon,
+            ToggleNavOpenIcon,
+            ToggleNavClosedIcon,
+            NavLinks,
+            NavButtonContainer,
+        },
+        props: {
+            userType: {
+                type: String,
+                default: 'public', //Others: client, employee
```

Comment:
You can add prop `validator` here with check for proper values.

Prop validations are mostly liked for thing like this - they inform developer what values he can use quickly : )

Reply:
Thanks for the tip!
Haven't reached this section in the docs yet.. :)